manim/mobject/vector_field.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import annotations
 
+from manim.mobject.graphing.coordinate_systems import CoordinateSystem
```

Comment:
Is there a reason why this import is not grouped with the other manim imports?

Reply:
nope, just auto import doing it's thing